util/string.go

```diff
@@ -87,7 +87,7 @@ func GetOwnerAndNameFromIdWithPanic(id string) (string, string) {
 	return tokens[0], tokens[1]
 }
 
-func GetOwnerAndNameFromId(id string) (string, string, error) {
+func GetOwnerAndNameFromUsernameWithOrg(id string) (string, string, error) {
```

Comment:
this func is used not only users id but for another entity ids too, (roleOwner/roleName = roleId  etc .. ), so "FromUsername" is wrong here.